v63004/u_inref/gtm3146.csh

```diff
@@ -0,0 +1,59 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+echo '# Testing '
+
+echo "# Create a 1 region DB region DEFAULT"
+echo ''
+$gtm_tst/com/dbcreate.csh mumps 1 >>& dbcreate_log.txt
+if ($status) then
+	echo "DB Create Failed, Output Below"
+	cat dbcreate_log.txt
+endif
+
+echo "# Create a file, shell.csh, to be sourced by child shells : "
+
+echo '#\!/usr/local/bin/tcsh -f' 		> "shell.csh"
+echo 'set echo ; set verbose'			>> "shell.csh"
+echo 'alias cp "jakewashere"' 			>> "shell.csh"
+echo 'if ( "$1" == "-c") then' 			>> "shell.csh"
+echo '	shift ' 				>> "shell.csh"
+echo 'endif' 					>> "shell.csh"
+echo 'echo "$*" > script.csh'			>> "shell.csh"
+echo 'source script.csh'			>> "shell.csh"
+echo ''
+
+chmod +x ./shell.csh
+chmod +rw ./
+
+echo 'cat shell.csh'
+cat ./shell.csh
+echo ''
+
+echo '# Set $SHELL env var to have child shell shell.csh '
+setenv SHELL "$PWD/shell.csh"
+echo ''
+
+echo "# Running Mupip with "'$SHELL'" ./shell.csh"
+$MUPIP BACKUP "*" tstBackup >>& backup.log
```

Comment:
Check $status after MUPIP BACKUP to ensure it is 0. If not, print an error.

Reply:
A $status check block, mirroring those used after dbcreate and dbcheck, has been added after the $MUPIP BACKUP call.